api/client/stack/tasks.go

```diff
@@ -43,9 +45,20 @@ func newTasksCommand(dockerCli *client.DockerCli) *cobra.Command {
 }
 
 func runTasks(dockerCli *client.DockerCli, opts tasksOptions) error {
+	namespace := opts.namespace
 	client := dockerCli.Client()
 	ctx := context.Background()
 
+	services, err := getServices(ctx, client, namespace)
+	if err != nil {
+		return err
+	}
+
+	if len(services) == 0 {
+		fmt.Fprintf(dockerCli.Out(), "Nothing found in stack: %s\n", namespace)
+		return nil
+	}
```

Comment:
In this case I don't think we should add another API call.

We could add the warning if `len(tasks) == 0`, or we could just not warn.